### DISCUSSION TITLE
v7 How can I get the labels from the command line e.g. stage=something

### DISCUSSION THREAD

**Discussion by digistorm-developer** [deployphp/deployer#3083] [Help needed] [answered]
I can't figure out how to directly access the deployer command line arguments.
Example command: `vendor/bin/deployer.phar build stage=whatever`
Previous versions we could do `input()->getArgument('stage')`, but now that doesn't work. How can I get this value directly in deploy.php?

**Comment by digistorm-developer** [comment, marked as answer]
Actually, I just saw in another post here, that you can do `get('labels')['stage']` so all good!

**Reply by antonmedv** [reply]
This is a correct solution. Make sure what all of your hosts have defined label stage. 

Why labels in selector not accessible directly? Consider next CLI examples of selectors:
- all
- stage!=prod

what stage value should be in this case?
